server/middlewares/render/bundle.tsx

```diff
@@ -0,0 +1,64 @@
+import { Request } from 'express';
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import { Provider } from 'react-redux';
+import { StaticRouter } from 'react-router-dom/server';
+import { I18nextProvider } from 'react-i18next';
+
+import { isDev } from '../../../lib/env';
+import { store } from '@redux/store';
+
+const getBundleHtml = (req: Request) => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const App = require(`../../../${isDev ? 'dist/' : ''}ssr.bundle.js`).App;
+
+  return renderToString(
+    <I18nextProvider i18n={req.i18n}>
+      <Provider store={store}>
+        <StaticRouter location={req.url}>
+          <App />
+        </StaticRouter>
+      </Provider>
+    </I18nextProvider>
+  );
+};
+
+export const getPageHtml = (req: Request) => {
+  const bundleHtml = getBundleHtml(req);
+
+  const initialLanguage = req.i18n.languages[0];
+  const initialStore: any = {};
```

Comment:
❗️
Лучше более специфичное имя подобрать, потому что в таком виде больше отсылает к `redux`-стору
```suggestion
  const initialLanguagesStore: any = {}; // initialLanguagesConfig
```

Reply:
Да, тут надо поменять. Это изменение тоже в следующем ПР сделаю.